modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/sync/SyncAccountManager.kt

```diff
@@ -77,14 +75,13 @@ class SyncAccountManager @Inject constructor(
                 val token = bundle.getString(AccountManager.KEY_AUTHTOKEN)
                 // Token failed to refresh
                 if (token == null) {
-                    val intent = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
-                        bundle.getParcelable(AccountManager.KEY_INTENT, Intent::class.java)
+                    val intent = BundleCompat.getParcelable(bundle, AccountManager.KEY_INTENT, Intent::class.java)
+                    if (intent == null) {
+                        throw NetworkErrorException()
                     } else {
-                        @Suppress("DEPRECATION")
-                        bundle.getParcelable(AccountManager.KEY_INTENT) as? Intent
```

Comment:
There is an Android KTX `BundleCompat` to reduce this logic. I will add it to other areas of the code in a separate PR. https://developer.android.com/reference/kotlin/androidx/core/os/BundleCompat